src/simulator/views/map/map_view.py

```diff
@@ -294,20 +305,26 @@ def HDScreenShot(self):
         cam.getLens().setAspectRatio(width / height)
         npCam = NodePath(cam)
         npCam.reparentTo(self.__world)
-        x,y,z = self.__world.getPos()
-        npCam.setPos(x,y+0.5,z+77.3)
-        npCam.setP(-90)
+        x, y, z = self.__world.getPos()
+        # 3d case
+        if c > 1:
+            npCam.setPos(x-a * 2.1, y - max_height * 3.6, z)
+            npCam.setH(-30)
+        else:
+            npCam.setPos(x, y, z + max_width * 2 + c)
```

Comment:
+c seems a little extraneous here, if we know it's zero?

Reply:
`c` is 1 in this case, but yh maybe best to remove it. @rm1819

Reply:
Cleaning this up now

Reply:
Done